contrib/docs/tutorials/authenticate-api-requests.md

```diff
@@ -129,20 +129,28 @@ function msUntilExpiry(token: string): number {
 
 // Calls the specified url regularly using an auth token to set a token cookie
 // to authorize regular HTTP requests when loading techdocs
-async function setTokenCookie(url: string, getIdToken: () => Promise<string>) {
-  const token = await getIdToken();
+async function setTokenCookie(url: string, identityApi: IdentityApi) {
+  const { token } = await identityApi.getCredentials();
+  if (!token) {
+    // In some cases, the token might be undefined - we will wait 10 seconds
+    // to try this again.
```

Comment:
Uh, that seems like a bug, when does this happen? x)

Reply:
I haven't had this myself - but on my last PR @freben wrote:

"and be prepared to deal with that token might end up being undefined; if it is, it can just try again 10 seconds later or whatever."

I might have not understood this correctly?

Reply:
Hmm, it's possible to get an `undefined` token, but it's not a temporary state. It happens when you don't have a verified identity, which really would be a configuration error if authentication is required. I think it probably makes the most sense to simply skip the cookie request if the token is undefined.